config/rbac/role.yaml

```diff
@@ -6,10 +6,20 @@ metadata:
   name: manager-role
 rules:
 - apiGroups:
-  - coordination.k8s.io
   - ""
   resources:
-  - configmaps
+  - leases
```

Comment:
This should be the other way around - leases not configmaps right?

Reply:
Good catch, @amuraru . Updated.